packages/plugin-udio/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@elizaos/plugin-udio",
+    "version": "0.1.0",
+    "description": "Udio AI Music Generation Plugin for Eliza",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "scripts": {
+        "build": "tsc",
+        "test": "jest",
+        "lint": "eslint src/**/*.ts"
+    },
+    "dependencies": {
+        "@elizaos/eliza": "^0.1.0",
+        "udio_wrapper": "^1.0.0"

```

Comment:
Hi @Freytes I'm not able to locate this dependency or install locally, any ideas? 
https://www.npmjs.com/search?q=udio_wrapper